src/metabase/lib/native.cljc

```diff
@@ -260,3 +260,10 @@
   [query :- ::lib.schema/query]
   (assert-native-query! (lib.util/query-stage query 0))
   (:engine (lib.metadata/database query)))
+
+(defmethod lib.query/can-save-method :mbql.stage/native
+  [query]
+  (every? (fn [[_k {:keys [required, default]}]]
+            (or (not required)
+                (not-empty (if (vector? default) (first default) default))))
```

Comment:
It's not clear to me at a glance why this unwrapping is necessary. I saw it in the TS code too, but I don't know the tag structure that well.

Can you add a comment showing the different cases we need to check for? (Alternatively, split that logic out into `(has-default? template-tag)` and document it there.)